src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -2,76 +2,84 @@ import React from 'react';
 import SearchResults from '../containers/SearchResults';
 
-export default function() {
-  return (
-    <>
-      <SearchForm />
-      <SearchResults />
-      <h2>Top tasks for frequently downloaded VA forms</h2>
-      <p>
-        You can now do many form tasks online, like filing a disability claim or
-        applying for the GI Bill. Get started online, and we’ll walk you through
-        step-by-step.
-      </p>
-      <ul className="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row">
-        <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
-          <b>File a VA disability claim</b>
-          <hr className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary" />
-          <p className="va-nav-linkslist-description">
-            Equal to VA Form 21-526EZ
-          </p>
-          <a
-            className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
-            href="/disability/how-to-file-claim/"
-          >
-            <span>
-              Read more
-              <span className="vads-u-visibility--screen-reader">
-                about filing a VA disability claim
-              </span>
-              <i className="fa fa-chevron-right vads-facility-hub-cta-arrow" />
+const onFeaturedContentClick = header => () => {
+  window.dataLayer.push({
+    event: 'nav-featured-content-link-click',
+    'featured-content-header': header,
+  });
+};
+
+export default () => (
+  <>
+    <SearchForm />
+    <SearchResults />
+    <h2>Top tasks for frequently downloaded VA forms</h2>
+    <p>
+      You can now do many form tasks online, like filing a disability claim or
+      applying for the GI Bill. Get started online, and we’ll walk you through
+      step-by-step.
+    </p>
+    <ul className="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row">
+      <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
+        <b>File a VA disability claim</b>
+        <hr className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary" />
+        <p className="va-nav-linkslist-description">
+          Equal to VA Form 21-526EZ
+        </p>
+        <a
+          className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
+          href="/disability/how-to-file-claim/"
+          onClick={onFeaturedContentClick('File a VA disability claim')}
```

Comment:
I just added it as an `onClick` handler for the a tags, but lmk if it should be implemented differently! 🙂@ncksllvn